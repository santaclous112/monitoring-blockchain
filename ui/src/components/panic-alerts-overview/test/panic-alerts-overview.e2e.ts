import { newE2EPage } from '@stencil/core/testing';

describe('panic-alerts-overview', () => {
    it('renders', async () => {
        const page = await newE2EPage();
        await page.setContent('<panic-alerts-overview></panic-alerts-overview>');

        const element = await page.find('panic-alerts-overview');
        expect(element).toMatchSnapshot();
    });
});
