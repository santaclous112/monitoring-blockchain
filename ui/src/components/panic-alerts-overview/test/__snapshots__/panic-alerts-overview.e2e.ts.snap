// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`panic-alerts-overview renders 1`] = `
<panic-alerts-overview>
  <h1 class="panic-alerts-overview__title" id="alerts-overview">
    ALERTS OVERVIEW
  </h1>
  <svc-surface>
    <div class="panic-alerts-overview__data-table-container">
      <svc-filter debounce="100" event-name="filter-changed">
        <div class="panic-alerts-overview__slots">
          <svc-select header="Select chains" id="chains-filter" multiple="" name="selected-chains" placeholder="Select chains"></svc-select>
          <svc-select header="Select severities" id="severity-filter" multiple="" name="selected-severities" placeholder="Select severities"></svc-select>
        </div>
        <div class="panic-alerts-overview__slots">
          <svc-select header="Select sources" id="sources-filter" multiple="" name="selected-sources" placeholder="Select sources"></svc-select>
          <div class="panic-alerts-overview__date-time-container">
            <h4>
              From:
            </h4>
            <svc-date-time mode="datetime-local" name="date-time-from" value=""></svc-date-time>
            <h4>
              To:
            </h4>
            <svc-date-time mode="datetime-local" name="date-time-to" value=""></svc-date-time>
          </div>
        </div>
      </svc-filter>
      <svc-data-table last-clicked-column-index="1" no-records-message="There are no alerts to display at this time" ordering="descending"></svc-data-table>
    </div>
  </svc-surface>
</panic-alerts-overview>
`;
